README.md

```diff
@@ -2,8 +2,30 @@
 
 For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
 
-
-
 --- 
 
 This portfolio project was created as part of a Digital Skills Bootcamp in Software Engineering provided by [Northcoders](https://northcoders.com/)
+
+___
+
+Setting Up Environment Variables
+
+To run this project locally, you need to create two environment variable files, .env.test and .env.development, in the root of your project directory. These files will contain the database names required to connect to the respective environments.
+
+Steps to Create Environment Variable Files:
+
```

Comment:
The idea behind using the .env files and gitignoring them is so that we don't have our database names here - we should instead tell users how to find the database names (in setup.sql) 
(In reality, we also would not push up setup.sql as it exposes the database names too but in this instance we can't avoid it as users will need it here!)